builds/assets/lang/enAU.json

```diff
@@ -702,6 +702,8 @@
   "toolFilterCylinderContinue": "Select a second point to complete the filter",
   "toolCrossSectionStart": "Select a point on one edge of the cross section to begin...",
   "toolCrossSectionContinue": "Select a second point to set the orientation and width of the cross section",
+  "supportLoginPrefix": "Having trouble logging in? Visit ",
+  "supportConvertingPrefix": "Having trouble with converting? Visit ",
```

Comment:
This needs to use the `vcStringFormat` system, other languages might not have the URL at the end of the string. Use `{0}` as a marker for where the URL needs to be placed - there's plenty of examples in this file that you can use to see how it's done elsewhere.